lua/metals/util.lua

```diff
@@ -120,4 +120,30 @@ M.path = (function()
   }
 end)()
 
+---- UI. Probably this should be a separate ui.lua module if this grows.
+---- CMD based UI:
+M.input_box = function(prompt)
+  return vim.fn.input(prompt)
+end
+
+M.input_list = function( options)
+  return vim.fn.inputlist(options)
+end
+---------------------------------------------------------------------
+
+-- TODO: See if nvim provides similar debugging function
+M.to_string = function(o)
```

Comment:
More of a general question due to my lack of lua knowledge, but how are functions like this normally handled? I see this want like a `private[metals]`, but I'm assuming there isn't one? So anyone using the plugin can reach into `util` and use this if they want right?

Reply:
This is my first lua code so I cannot answer.Yes, adding it to the module we can call it from outside util. Also if make it local the linter will complain that it isn't used, if it's exported in the module there is no warning even if not used anywhere.

Reply:
No worries. This is fine as is, I was just curious.